platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/util/BootstrapUtils.java

```diff
@@ -168,8 +175,34 @@ private BootstrapUtils() {}
     public static @NonNull Configuration loadConfig(
             @NonNull final PathsConfig pathsConfig, @NonNull final Map<NodeId, SwirldMain> appMains)
             throws IOException {
-        Objects.requireNonNull(pathsConfig, "pathsConfig must not be null");
         Objects.requireNonNull(appMains, "appMains must not be null");
+        final var configurationBuilder = ConfigurationBuilder.create();
```

Comment:
The platform does not currently use `var` anywhere. Not worth holding up the PR though.